README.md

```diff
@@ -1 +1,134 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Hello World</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Hello_World] <a name="about-project"></a>
+The goal of this project master the use of linters, gitignore and best practices in creating a README file and making a professional pull request. No complex coding is used in this project.
+
+## 🛠 Built With <a name="built-with"></a>
+1- HTML\
+2- CSS\
+3- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1- Web browser\
+2- Code editor\
+3- Git-smc
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command:
+
+```sh
+  cd my-folder
+  git clone https://github.com/Wineshuga/Hello-world.git
+```
+
+### Install
+
+Install this project with:
+
+> No installation required
+
+### Usage
+
+To run the project: 
+
+Open index.html usig live server
+
+### Run tests
+
+To run tests, run the following command:
+
+> Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 Nweneary Uzochukwu Winnie
+
+- GitHub: [@githubhandle](https://github.com/wineshuga)
+- Twitter: [@twitterhandle](https://twitter.com/wineshuga)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/wineshuga)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+- <i>pending</i>
```

Comment:
[Optional] Well done on adding the 'Future Features' section heading!👍 However, it would be god to add "future features" and include 1-3 potential features in this section. Remember in this section, you should outline your plans for the future development of the project. Include details about the features, functionalities, or improvements you intend to add. It should demonstrate your vision and give readers an idea of what to expect. Keep up the good work!

Please include 1-3 potential fractures in this section.
| Your        | Professional |
|---------------------|-----------------------|
| ![image](https://github.com/Wineshuga/Hello-world/assets/55840999/be230e80-64b9-4079-b9a0-6592ac7a35ed) | ![image](https://github.com/2004-okari/HELLO_MICROVERSE/assets/55840999/6c06d9c5-1153-4928-bfd9-d55057f77c97) |


Example feature:

- I will deploy this project